tools/sql/main.go

```diff
@@ -55,140 +58,155 @@ func BuildCLIOptions() *cli.App {
-			Value:  "",
-			Usage:  "password used for authentication when connecting to sql host",
-			EnvVar: "SQL_PASSWORD",
-		},
-		cli.StringFlag{
-			Name:   schema.CLIFlagDatabase,
-			Value:  "cadence",
-			Usage:  "name of the sql database",
-			EnvVar: "SQL_DATABASE",
-		},
-		cli.StringFlag{
-			Name:   schema.CLIFlagPluginName,
-			Value:  "mysql",
-			Usage:  "name of the sql plugin",
-			EnvVar: "SQL_PLUGIN",
-		},
-		cli.BoolFlag{
-			Name:  schema.CLIFlagQuiet,
-			Usage: "Don't set exit status to 1 on error",
-		},
-		cli.GenericFlag{
-			Name:   schema.CLIFlagConnectAttributes,
-			Value:  &cliflag.StringMap{},
-			Usage:  "sql connect attributes (must be in key1=value1,key2=value2,...,keyN=valueN format, e.g. cluster=dca or cluster=dca,instance=cadence)",
-			EnvVar: "SQL_CONNECT_ATTRIBUTES",
-		},
-		cli.BoolFlag{
-			Name:   schema.CLIFlagEnableTLS,
-			Usage:  "enable TLS over sql connection",
-			EnvVar: "SQL_TLS",
-		},
-		cli.StringFlag{
-			Name:   schema.CLIFlagTLSCertFile,
-			Usage:  "sql tls client cert path (tls must be enabled)",
-			EnvVar: "SQL_TLS_CERT_FILE",
-		},
-		cli.StringFlag{
-			Name:   schema.CLIFlagTLSKeyFile,
-			Usage:  "sql tls client key path (tls must be enabled)",
-			EnvVar: "SQL_TLS_KEY_FILE",
-		},
-		cli.StringFlag{
-			Name:   schema.CLIFlagTLSCaFile,
-			Usage:  "sql tls client ca file (tls must be enabled)",
-			EnvVar: "SQL_TLS_CA_FILE",
-		},
-		cli.BoolFlag{
-			Name:   schema.CLIFlagTLSEnableHostVerification,
-			Usage:  "sql tls verify hostname and server cert (tls must be enabled)",
-			EnvVar: "SQL_TLS_ENABLE_HOST_VERIFICATION",
+		&cli.StringFlag{
+			Name:    schema.CLIFlagEndpoint,
+			Aliases: []string{"ep"},
+			Value:   "127.0.0.1",
+			Usage:   "hostname or ip address of sql host to connect to",
+			EnvVars: []string{"SQL_HOST"},
+		},
+		&cli.IntFlag{
+			Name:    schema.CLIFlagPort,
+			Aliases: []string{"p"},
+			Value:   defaultSQLPort,
+			Usage:   "port of sql host to connect to",
+			EnvVars: []string{"SQL_PORT"},
+		},
+		&cli.StringFlag{
+			Name:    schema.CLIFlagUser,
+			Aliases: []string{"u"},
+			Value:   "",
+			Usage:   "user name used for authentication when connecting to sql host",
+			EnvVars: []string{"SQL_USER"},
+		},
+		&cli.StringFlag{
+			Name:    schema.CLIFlagPassword,
+			Aliases: []string{"pw"},
+			Value:   "",
+			Usage:   "password used for authentication when connecting to sql host",
+			EnvVars: []string{"SQL_PASSWORD"},
+		},
+		&cli.StringFlag{
+			Name:    schema.CLIFlagDatabase,
+			Aliases: []string{"db"},
+			Value:   "cadence",
+			Usage:   "name of the sql database",
+			EnvVars: []string{"SQL_DATABASE"},
+		},
+		&cli.StringFlag{
+			Name:    schema.CLIFlagPluginName,
+			Aliases: []string{"pl"},
+			Value:   "mysql",
+			Usage:   "name of the sql plugin",
+			EnvVars: []string{"SQL_PLUGIN"},
+		},
+		&cli.BoolFlag{
+			Name:    schema.CLIFlagQuiet,
+			Aliases: []string{"q"},
+			Usage:   "Don't set exit status to 1 on error",
```

Comment:
this is definitely not what this flag means to me, but it's the existing behaviour, so we should preserve it. This is probably the thing used in someone's build pipeline to ensure the schema is working.

Reply:
fixed